charts/datadog-monitors/values.yaml

```diff
@@ -0,0 +1,83 @@
+# -- (`string`) Optional shared pagerduty service name for monitors, will turn to a tag for alerts - if not provided, the .Release.name will be used by default
+serviceName:
+# -- (`string`) Optional shared pagerduty team name for monitors, will turn to a tag for alerts - if not provided, the tag will not be added
+team:
+
+# -- (`map[string]interface{}`) List of monitors
+monitors:
+  # -- (`map[string]interface{}`) Required: monitor resource name, Required unique monitor resource name(needed to allow value overrides and used a datadog monitor resource name)
+  resourceName:
+    # -- (`boolean`) Optional: whether to exclude the monitor, defaults to False
+    disabled: true
+    # -- (`string`) Require: monitor name
+    name: "[kubernetes] Monitor Kubernetes Failed Pods in Namespaces"
+    # -- (`string`) Required: monitor message
+    message: "More than ten pods are failing in ({{kube_cluster_name.name}} cluster). \n The threshold of ten pods varies depending on your infrastructure. Change the threshold to suit your needs."
+    # -- (`string`) Optional: monitor piority
+    priority: "2"
+    # -- (`string`) Required: monitor query
+    query: "change(avg(last_5m),last_5m):sum:kubernetes_state.pod.status_phase{phase:failed} by {kube_cluster_name,kube_namespace} > 10"
+    # -- (`string`) Optional: monitor type, if not specified will default to 'query alert'
+    # Datadog monitor types to type values mapping:
+    # - anomaly: `query alert`
+    # - APM: `query alert` or `trace-analytics alert`
+    # - composite: `composite`
+    # - custom: `service check`
+    # - forecast: `query alert`
+    # - host: `service check`
+    # - integration: `query alert` or `service check`
+    # - live process: `process alert`
+    # - logs: `log alert`
+    # - metric: `query alert`
+    # - network: `service check`
+    # - outlier: `query alert`
+    # - process: `service check`
+    # - rum: `rum alert`
+    # - SLO: `slo alert`
+    # - watchdog: `event-v2 alert`
+    # - event-v2: `event-v2 alert`
+    # - audit: `audit alert`
+    # - error-tracking: `error-tracking alert`
+    # - database-monitoring: `database-monitoring alert`
+    # - network-performance: `network-performance alert`
+    # - service-discovery: `service-discovery alert`
+    type: "query alert"
+    # -- (`string[]`) Optional: Additional monitor tags(will be added on top of the default tags:service, team, namespace, cluster, app_env, app_group)
+    tags:
+      - 'tagname:tagvalue'
+    # -- (`map[string]interface{}`) Optional: monitor options
+    options:
```

Comment:
I would ignore all the sub-keys here and just point to the upstream documentation instead as we are not likely to keep this list up to date.

Reply:
It's an option I thought about for the whole thing, in theory the names and the structure is exactly the same as the CRD - I could in theory point out to the CRD(though some of the info was not properly documented and I had to extract it from the code).